port.h

```diff
@@ -90,4 +90,19 @@ SOFTWARE.
 #error "Unknown architecture."
 #endif
 
+#ifdef BRICKS_WINDOWS
+
+// #include <windows.h>  -- `#include <windows.h>` does not play well with WSA -- D.K.
+
+// These two headers seem to work fine for both WSA and `FindFirst*` files.
+#include <WS2tcpip.h>
+#include <corecrt_io.h>
+
+// Visual Studio does not define `M_PI`.
```

Comment:
It's more of me being lazy to not rely on a non-standard `M_PI` in the demo. Added issue https://github.com/KnowSheet/Bricks/issues/92